docs/workshops/kubecon_azure_track.md

```diff
@@ -0,0 +1,114 @@
+Table of contents:
+- [Overview](#overview)
+- [Basic Cheet Sheet](#basic-cheet-sheet)
+- [Adjustments](#adjustments)
+    - [Step: Install Karpenter](#step-install-karpenter)
+    - [Step: Basic NodePool](#step-basic-nodepool)
+        - [Step: Scaling Application](#step-scaling-application)
+        - [Step: Limit Resources](#step-limit-resources)
+        - [Step: Disruption](#step-disruption)
+        - [Step: Drift](#step-drift)
+        - [Step: RightSizing](#step-rightsizing)
+    - [Step: Consolidation](#step-consolidation)
+        - [Step: Single Node Consolidation](#step-single-node-consolidation)
+        - [Step: Multi Node Consolidation](#step-multi-node-consolidation)
+
+## Overview
+
+This document details the steps to follow the KubeCon workshop using an AKS cluster in Azure.
+
+To follow along using this workshop, simply go through the steps detailed in this document. For each step of the workshop, since Karpenter is built in the open source, and has a lot of cloud agnostic elements, there is a link to the AWS version (in the step's header), as much of the AWS workshop will translate directly over to Azure. However, for any steps where it diviates, there will be a few important notes, and adjustments detailed under those steps.
+
+## Basic Cheet Sheet
+
+When you see `aws-node-viewer` use `aks-node-viewer` instead.
+
+## Adjusted Instructions
+
+### Step: [Install Karpenter](https://catalog.workshops.aws/karpenter/en-US/install-karpenter)
+
+- Instead follow [1_install_karpenter.md](https://github.com/Azure/karpenter-provider-azure/tree/main/docs/workshops/1_install_karpenter.md)
+
+### Step: [Basic NodePool](https://catalog.workshops.aws/karpenter/en-US/basic-nodepool)
+
+- Notes:
+    - AKSNodeClass is Azure’s equivalence to EC2NodeClass for Azure specific settings. Each Karpenter NodePool must contain a reference to an AKSNodeClass via the spec.template.spec.nodeClassRef.
+
+- Adjustments:
+    - The same concepts within the workshop generally translate to AKS. However, for the actual deployment step, we need a `AKSNodeClass`, and a few additional Azure specific adjustments. So, instead of the given deployment command follow [2_basic_noodpool.md](https://github.com/Azure/karpenter-provider-azure/tree/main/docs/workshops/2_basic_noodpool.md) 
+
+### Step: [Scaling Application](https://catalog.workshops.aws/karpenter/en-US/basic-nodepool/scaling)
+
+- Notes:
+    - The creation of the node and nodeclaim by karpenter might take ~90s. However, you can confirm the node claims creation in the `k8s` api beforehand using the command:
+        ```bash
+        kubectl get nodeclaims -A
```

Comment:
No need for `-A`, NodeClaims are not namespaced
(Also, `kubectl get karpenter` is pretty convenient ...)

Reply:
Removed the `-A`